topics/admin/tutorials/poll-ssa/slides.html

```diff
@@ -0,0 +1,415 @@
+Managing Conda envs for tools within Galaxy, doesn’t take extra time compared to outside of Galaxy – I initially thought it might.
+]
+
+---
+
+
+### DB Schema Update Frequency
+
+
+.pull-left[![DB Schema Update Frequency](../../images/poll-ssa/16-DB-Schema-Update-Frequency.jpg)]
+
+
+.pull-right[
+Happens on average once a year
+]
+
+
+---
+
+
+### DB Schema Update Duration
+
+
+.pull-left[![DB Schema Update Duration](../../images/poll-ssa/17-DB-Schema-Update-Duration.jpg)]
+
+
+.pull-right[
+Is very quick and transparent (assuming Ansible usage)
+]
+
+
+---
+
+
+### Ansible
+
+
+.pull-left[![Galaxy Upgrades Ansible](../../images/poll-ssa/18-Galaxy-Upgrades-Ansible.png)]
+
+
+.pull-right[
+Is a highly-used (about 80%) tool that, according to the other sections, greatly simplifies admin.
+]
+
+
+---
+
+
+### Current Version & Upgrade Frequency
+
+
+.pull-left-large[
+.image-45[![Galaxy Upgrades Current Version](../../images/poll-ssa/19-Galaxy-Upgrades-Current-Version.jpg)]
+.image-45[![Galaxy Upgrades Frequency](../../images/poll-ssa/20-Galaxy-Upgrades-Frequency.jpg)]
+]
+
+
+.pull-right-small[
+- Half the respondents are able to continue using a version that is 2-3 years old
+- This is not recommended as versions older than a year do not receive security updates
+- But most update about once a year
+]
+
+
+---
+
+
+### Galaxy Upgrades Duration (excluding DB schema migration duration)
+
+
+.pull-left[![Galaxy Upgrades Duration](../../images/poll-ssa/21-Galaxy-Upgrades-Duration.jpg)]
+
+
+.pull-right[
+- Avg 3 hours, big variance, which doesn’t seem proportional to the amount of users, but rather other factors
+- Some updates are very simple, especially if you use Ansible 
+- Some non-backwards-compatible changes like uWSGI to Gunicorn take longer
+- Some admins take longer because of custom plugins (that need to be updated) or test deployment environments that need to be maintained.
+]
+
+
+---
+
+
+### DB Backup Frequency
+
+
+.pull-left[![DB Backup Frequency](../../images/poll-ssa/22-DB-Backup-Frequency.jpg)]
+
+
+.pull-right[
+- Although not necessarily Galaxy related, I was curious about the best practices
+- Avg. is 1 time per week, and instantaneous
+- Ansible role: https://galaxy.ansible.com/ui/standalone/roles/galaxyproject/postgresql/
+- General Galaxy DB Admin tutorial: https://training.galaxyproject.org/training-material/topics/admin/tutorials/backup-cleanup/tutorial.html
+]
+
+
+---
+
+
+### Intentional Restarts Frequency
+
+
+.pull-left[![Intentional Restarts Frequency](../../images/poll-ssa/23-Intentional-Restarts-Frequency.jpg)]
+
+
+.pull-right[
+
+**Context**
+- I had assumed restarts would heavily impact users, hence this question
+- But actually, restarts shouldn't interrupt existing jobs (be they local or on-cluster)
+
+**Analysis**
+- About once a month, although one admin manages to space it out to every 3 months
+- For applying system updates or adding new tools (rare bug, normally not necessary to restart for this)
```

Comment:
```suggestion
- If you configure Galaxy to use a job scheduler (SLURM, HTCondor, etc), restarts will not interrupt existing jobs.
- Generally admins restart about once a month, in order to apply system updates or reconfigure Galaxy or its tools
```

Reply:
Good summarising.

I think I will add another line saying some of the config/tool changes can be done without restarting (although I'm not an expert, it's what I understood)

Reply:
Sentence added here https://github.com/galaxyproject/training-material/pull/4972/commits/63746ecefbed9613de544db515dc9b5152d3888c